chart-templates/crd-base/Chart.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: ${api_version}
+version: ${chart_version}
+description: A Rancher chart that creates ${name} CRDs within a cluster.
```

Probably need to change `A Rancher chart`. I think @cbron mentioned this in slack.

---

Good catch, will update this in the next commit:
```suggestion
description: Installs the CRDs for ${name}.
```